hooks/queries/user/useUserVotingAndStakingDetails.ts

```diff
@@ -13,7 +13,7 @@ export function useUserVotingAndStakingDetails() {
     () => getUserData(address),
     {
       enabled: !!address,
-      refetchInterval: (data) => (data ? false : 100),
+      refetchInterval: oneMinute,
```

Comment:
since hitting the graph is cheap, we might as well poll this in case its changed from actions taken elsewhere